README.md

```diff
@@ -4,3 +4,12 @@ This repository contains our shared GitHub Actions, written in Go,
 that are used by [FerretDB](https://github.com/FerretDB/FerretDB),
 [dance](https://github.com/FerretDB/dance),
 and other repositories.
+
+## Contributing
+
+* To contribute to this project, please read the [CONTRIBUTING.md](https://github.com/FerretDB/github-actions/blob/main/CONTRIBUTING.md) 
+and [CODE_OF_CONDUCT.md](https://github.com/FerretDB/FerretDB/blob/main/CODE_OF_CONDUCT.md) guideline to know more.
+* Is everything working as it should?
+If not, please let us know by [creating an issue](https://github.com/FerretDB/github-actions/issues/new/choose).
+You can create issues for bugs, documentation, features, or enhancements.
```

Comment:
Unlike FerretDB, we don't have a wizard to select issue type, and external people can't assign labels